packages/core/src/create-lighthouse-benchmark.ts

```
@@ -31,6 +31,26 @@ async function runLighthouse(
     `${namePrefix}_performance_profile.json`,
     JSON.stringify(runnerResult.artifacts)
   );
+  if (runnerResult.lhr.runtimeError) {
```

Ignoring errors in perf tests is a reliable way to be confused. 
While working on [Popmenu CI perf tests](https://github.com/Popmenu/popmenu/pull/6480) I stumbled on some `unexpected improvements` that actually were `errors on the page` and spent some time scratching my head.
Failing CI jobs is much better.